### DISCUSSION TITLE
How to run the HelloMaui WinUI project

### DISCUSSION THREAD

**Discussion by williambuchanan2** [dotnet/maui#1851] [Q&A]
Hi all

This might be a silly question, but I am stuck!

I installed the HelloMaui example project (dotnet new maui -n HelloMaui) and opened it in Visual Studio 2022. I managed to run the Android version but I can't get the Windows version to run.

I can see there is a HelloMaui.WinUI project - how do I execute it? When I try to run it I get an error "The project doesn't know how to run the profile HelloMaui.WinUI".

Thanks

**Comment by mcguu** [comment]
I can not run it too.

Android local device can not be found.

I can only run it in emulator ?

**Comment by newcc845** [comment]
Same here, can't run HelloMAUI WinUI project and successful for mobile (Android).

Anyone can give advise?

**Reply by danch15** [reply]
I use vs2022 priview3.1 and dotnet6 priview 7. I can debug the android app in the Android Emulator 30, but I can't install the "com.microsoft.hellomaui-Signed.apk" in my Samsung NOTE10(Android 10) or other android devices. It seems the problem is in the csproj's property "PlatformTarget", the default value is null, and you can select AnyCpu, x86 and x64, but I cannot find the arm, arm64. Somebody help me? Thanks.

**Reply by newcc845** [reply]
I still new to this, why u take the **signed.apk**? there is one without *signed* word.

**Comment by cosmoslx** [comment]
### I managed to run it:

> ![image](https://user-images.githubusercontent.com/673068/131801180-eb08438f-df2e-4291-b99d-9e2cd1805872.png)

### Follow these steps:

1. Install the [Single-project MSIX Packaging Tools for VS 2022](https://marketplace.visualstudio.com/items?itemName=ProjectReunion.MicrosoftSingleProjectMSIXPackagingToolsDev17)
2. Right click the WinUI Project and select the "deploy" operation to deploy the App
> ![image](https://user-images.githubusercontent.com/673068/131801847-61fabdab-7181-456a-b447-fd98a83603f0.png)
3. Launch the App from Start Menu
> ![image](https://user-images.githubusercontent.com/673068/131802148-b89139ee-55bd-4da3-a794-5c4ec106e8e3.png)

**Reply by newcc845** [reply]
Thank You, I was thought it can be generate as exe but never thought can deploy into windows.
